code/modules/events/immovable_rod.dm

```diff
@@ -35,12 +35,18 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	z_original = z
 	destination = end
 	notify_ghosts("\A [src] is inbound!",
-			enter_link="<a href=?src=[UID()];orbit=1>(Click to follow)</a>",
+			enter_link="<a href=?src=[UID()];follow=1>(Click to follow)</a>",
```

Comment:
NOTIFY_FOLLOW is for following as ghost when clicking the alert icon, enter_link prints to chatlog with a text link you can click to also follow